src/neo/Ledger/Blockchain.State.cs

```diff
@@ -0,0 +1,11 @@
+using Akka.Actor;
+using Neo.Network.P2P.Payloads;
+
+namespace Neo.Ledger
+{
+    public sealed partial class Blockchain : UntypedActor
+    {
+        public StateRoot LatestValidatorsStateRoot => new StateRoot();
```

Comment:
It's not good to split a new `Blockchain.State.cs` file with too little code.

Reply:
Agree